package.json

```diff
@@ -38,6 +41,7 @@
     "eslint-config-google": "0.14.0",
     "jest": "29.3.1",
     "mocha": "10.1.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
I think adding dependency just not to use basic && command feels overkill.

Reply:
Okay, let me make a commit to it.